### PR TITLE
Add resolve graph parsing and display

### PR DESCRIPTION
**Pull request by yageek**
In last Xcode version, if you use the swift package manager in Xcode, xcodebuild will print the resolved packages.

### CODE REVIEW

**Review by thii — COMMENTED** [review]
Can you add a dead simple test?

**Comment by yageek** [comment]
I added a test and fixed and thanks to it.

File: Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift [hunk 1]
```diff
@@ -320,4 +320,19 @@ final class XcbeautifyLibTests: XCTestCase {
...
+
+    func testPackageGraphResolved() {
+
+        // Start
+        let start = noColoredFormatted("Resolve Package Graph")
+        XCTAssertEqual(start, "Resolve Package Graph")
+
+        // Ended
+        let ended = noColoredFormatted("Resolved source packages:")
+        XCTAssertEqual(ended, "Resolved source packages")
```
Comment (thii) [inline Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift:332]: Should we include the trailing colon too?

**Review by thii — APPROVED** [review]

File: Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift (hunk 1 above)
Comment (yageek) [reply Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift:332]: `xcodebuild` produces a line with a colon at the end. Do you mean adding it to the formatted output?

Comment (thii) [reply Tests/XcbeautifyLibTests/XcbeautifyLibTests.swift:332]: Yes. There will be contents after this, right?

**Pull request merged**